### ISSUE TITLE
Why is the anchor string here not fully displayed?

### ISSUE THREAD

**Issue by iroan** [ExodiusStudios/vscode-comment-anchors#65]
# os:
```
➜  /etc cat lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.3 LTS"

```
# vscode:
```
Version: 1.39.2
Commit: 6ab598523be7a800d7f3eb4d92d7ab9a66069390
Date: 2019-10-15T15:33:40.634Z
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Linux x64 5.0.0-32-generic
```
# info
```
Name: Comment Anchors
Id: exodiusstudios.comment-anchors
Description: Place anchor tags within comments for easy file & workspace navigation.
Version: 1.6.1
Publisher: Exodius Studios
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ExodiusStudios.comment-anchors
```
# config in settings.json:
```
  "commentAnchors.workspace.enabled": false,
  "commentAnchors.workspace.lazyLoad": false,
  "commentAnchors.workspace.matchFiles": "**/*",
  "commentAnchors.parseDelay": 200,
  "commentAnchors.scrollPosition": "top",
  "commentAnchors.editorFolding": false,
  "commentAnchors.tagHighlights.enabled": true,
  "commentAnchors.tags.displayInSidebar": true,
  "commentAnchors.tags.displayInGutter": false,
  "commentAnchors.tags.separators": [" ", ":", "-"]
```

![image](https://user-images.githubusercontent.com/10843214/68944308-839af880-07e8-11ea-925a-eb544e2fcb34.png)

thank you!

**Comment by macjuul** [comment]
The regex currently in place of searching for anchors is only compatible with the latin alphabet. This will be changed and fixed in the next update.

A temporary fix until then is to surround your anchor message in doubles quotes

**Comment by macjuul** [comment]
Fixed as of 1.7.0
